pkg/karmadactl/cmdinit/cert/cert.go

```diff
@@ -309,6 +311,30 @@ func GenCerts(pkiPath string, etcdServerCertCfg, etcdClientCertCfg, karmadaCertC
 	return genEtcdCerts(pkiPath, etcdServerCertCfg, etcdClientCertCfg)
 }
 
+func getCertAndKey(caCertPath, caKeyPath string) (*x509.Certificate, *crypto.Signer, error) {
+	var caCert *x509.Certificate
+	var caKey *crypto.Signer
+	var err error
```

Comment:
may be this would be more simple:

```suggestion
func getCertAndKey(caCertPath, caKeyPath string) (caCert *x509.Certificate, caKey *crypto.Signer, err error) {
```

Reply:
Thanks, your version is better.